main.go

```diff
@@ -167,6 +168,8 @@ func main() { //nolint:funlen
 		os.Exit(1)
 	}
 
+	(&webhook.OpenDataHubWebhook{}).SetupWithManager(mgr)
```

Comment:
move this a bit up, maybe right after call to `ctrl.NewManager() `so it gets time for webhook to be accessible when `CreateDefaultDSCI` is called